core/src/cats/cats.h

```diff
@@ -42,7 +42,8 @@
 
 #include <string>
 #include <vector>
-template <typename T> class dlist;
+template <typename T>
+class dlist;
```

Comment:
a recent version of clang-format should format this it in a single line with our configuration. Maybe you need an update? You can always use the clang-format in devtools/clang-format